Common/src/Storage/ITaskTable.cs

```diff
@@ -266,13 +270,20 @@ public interface ITaskTable : IInitializable
 
   /// <summary>
   ///   Change the status of the task to succeeded
-  /// </summary>
-  /// <param name="taskId">Id of the task to tag as succeeded</param>
+  ///   Updates:
+  ///   - <see cref="TaskData.Status" />
+  ///   - <see cref="TaskData.StartDate" />
+  ///   - <see cref="TaskData.EndDate" />
+  ///   - <see cref="TaskData.CreationToEndDuration" />
+  ///   - <see cref="TaskData.ProcessingToEndDuration" />
+  ///   - <see cref="TaskData.Output" />
```

Comment:
This should most likely be in `<details>` instead of `<summary>`

Reply:
Details are not shown in the tooltip. I don't like this idea because I need this information

Reply:
Sorry, I mixed with `remarks` which is what Microsoft use. I checked and remarks work on Visual Studio.